### PR TITLE
Add linting configuration and fix linitng problems

### PR DESCRIPTION
**Pull request by wvffle**

### CODE REVIEW

File: music_service_async_interface/__init__.py [hunk 1]
```diff
@@ -112,11 +113,10 @@ async def get_file_url(self, *args, **kwargs) -> str:
         ...
 
     if AsyncSeekableHTTPFile is not None:
+
         async def get_async_file(self, filename: Optional[str] = None, *args, **kwargs) -> AsyncSeekableHTTPFile:
             return await AsyncSeekableHTTPFile.create(
-                await self.get_file_url(*args, **kwargs),
-                self.title if filename is None else filename,
-                self.sess.sess
+                await self.get_file_url(*args, **kwargs), self.title if filename is None else filename, self.sess.sess
```
Comment (JuniorJPDJ) [inline music_service_async_interface/__init__.py]: uhm, I liked previous split, it's much more readable

**Review by JuniorJPDJ — CHANGES_REQUESTED** [review]
The issues with this configuration are probably the same as in https://github.com/FUMR/tidal-async/pull/27.
Only one came up in this pull, but it will probably piss me off later XD
Tell me what do you think about things I stated in both issues please.

**Comment by JuniorJPDJ** [comment]
What do you think about creating template repository with this configs and some other project files like .gitignore, license, github-actions or poetry project.
It would make creating new repositories way easier.

File: music_service_async_interface/__init__.py (hunk 1 above)
Comment (JuniorJPDJ) [reply music_service_async_interface/__init__.py]: Ok, now I know we just need a trailing coma to make this work as I want it to.
Could you please add it to this PR?

**Comment by wvffle** [comment]
Yup I planned to do this today

**Review by JuniorJPDJ — COMMENTED** [review]
<3

**Pull request merged**